1.12/test/run

```diff
@@ -187,7 +186,7 @@ test_for_output()
   info "Testing URL $url with HTTP hostname ${host} produces output $expect"
 
   while [ $attempt -le $max_attempts ]; do
-    response=$(curl -s -H "Host: ${host}" ${url})
+    response=$(curl -is -H "Host: ${host}" ${url})
     echo "${response}"
     status=$?
```

Comment:
`$?` is return code of `echo` command.  Is this intended?

Reply:
Great catch! I didn't even see that. Will updated in a minute.